src/server/scripts/Spells/spell_evoker.cpp

```diff
@@ -94,8 +98,50 @@ class spell_evo_glide : public SpellScript
     }
 };
 
+// 361469 - Living Flame (Red)
+class spell_evo_living_flame : public SpellScript
+{
+    PrepareSpellScript(spell_evo_living_flame);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo ({ SPELL_EVOKER_LIVING_FLAME_DAMAGE, SPELL_EVOKER_LIVING_FLAME_HEAL, SPELL_EVOKER_ENERGIZING_FLAME });
+    }
+
+    void HandleHitTarget(SpellEffIndex /*effIndex*/)
+    {
+        Unit* caster = GetCaster();
+        Unit* hitUnit = GetHitUnit();
+        if (caster->IsFriendlyTo(hitUnit))
+            caster->CastSpell(hitUnit, SPELL_EVOKER_LIVING_FLAME_HEAL, true);
+        else
+            caster->CastSpell(hitUnit, SPELL_EVOKER_LIVING_FLAME_DAMAGE, true);
+    }
+
+    void HandleLaunchTarget(SpellEffIndex /*effIndex*/)
+    {
+        Unit* caster = GetCaster();
+        if (caster->IsFriendlyTo(GetHitUnit()))
+            return;
+
+        if (AuraEffect* auraEffect = caster->GetAuraEffect(SPELL_EVOKER_ENERGIZING_FLAME, EFFECT_0))
+        {
+            int32 manaCost = GetSpell()->GetPowerTypeCostAmount(POWER_MANA).value_or(0);
+            if (manaCost != 0)
+                GetCaster()->ModifyPower(POWER_MANA, CalculatePct(manaCost, auraEffect->GetAmount()));
```

Comment:
ModifyPower or EnergizeBySpell? (packet difference)

Reply:
no spell in sniff to energize

Reply:
its power update packet in sniff